src/loader/dbconnect.js

```diff
@@ -0,0 +1,16 @@
+const users = [];
+const accounts = [];
+const piggybanks = [];
+const transactions = [];
+
+function getUserByEmail(email) {
```

Comment:
functions with operates with user should be stored at user controller

Reply:
ok